src/ActivitiesSDKAdapter.js

```diff
@@ -127,4 +127,46 @@ export default class ActivitiesSDKAdapter extends ActivitiesAdapter {
 
     return action$;
   }
+
+  /**
+   * Posts an activity and returns an observable to the new activity data
+   *
+   * @param {object} activityData  Object populated with data used to post an activity
+   * @returns {Observable.<Activity>} Observable stream that emits data of the newly activity
+   */
+  postActivity(activityData) {
+    const card = activityData.card && [
+      {
+        contentType: 'application/vnd.microsoft.card.adaptive',
+        content: activityData.card,
+      },
+    ];
+    const newActivity$ = from(this.datasource.messages.create({
+      roomId: activityData.roomID,
+      text: activityData.text,
+      attachments: card || activityData.attachments,
+    })).pipe(
+      map(({
+        id,
+        roomId,
+        text,
+        personId,
+        attachments,
+        created,
+      }) => ({
+        ID: id,
+        roomID: roomId,
+        text,
+        personID: personId,
+        attachments,
+        created,
+      })),
+      catchError((err) => {
+        logger.error('ACTIVITY', undefined, 'postActivity()', `Unable to post an activity in room with id "${activityData.roomID}"`, err);
```

Comment:
Better to present the whole activity object that failed, not just the room id
[`Unable to post activity`, activity], err